README.md

```diff
@@ -58,6 +58,10 @@ A skeleton program would look somewhat like this (yes, pkcs#11 is verbose):
 
 Further examples are included in the tests.
 
+To expose PKCS#11 keys using the
+[crypto.Signer interface](https://golang.org/pkg/crypto/#Signer),
+please see [github.com/thalesignite/pkcs11](https://github.com/thalesignite/crypto11).
```

Comment:
these urls don't match?

Reply:
Oh, good spot, sorry. Hang on.

Reply:
Fixed. Still only on the first cup of tea of the morning l-)